src/containers.js

```diff
@@ -38,19 +42,69 @@ const update = async (fileName, fileContents, install = false) => {
     return false;
   }
 
-  // write temp file, validate and then overwrite
   if (!await validComposeFile(fileContents)) {
     throw new Error(`Invalid docker-compose yml for file ${fileName}`);
   }
 
+  if (backupIsEnabled()) {
+    await backupComposeFiles();
+  }
+
   await overwriteComposeFile(filePath, fileContents);
   await dockerComposeCli.pull(filePath);
 
   return true;
 };
 
+
+/**
+ * Generates a new name containing date of creation and random characters to avoid most collisions
+ * @returns {string} Name for folder e.g. 2023-07-17_131438_817_2Tjx
+ */
+const genBackupFolderName = () => {
+  const filefriendlyDatetime = new Date()
```

Comment:
```suggestion
  const fileFriendlyDateTime = new Date()
```
because js doesn't have types or reserved words for `filefriendly` and `datetime`, so we use [camelCase](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/#use-lowercamelcase-for-variables-properties-and-function-names)